css-images-4/Overview.bs

```diff
@@ -2011,7 +2011,9 @@ Color Stop “Fixup”</h4>
 		will result in a 100px red stripe and 200px green stripe,
 		giving red 1 share and green 2 shares of the 300px remaining
 		after subtracting blue's 100px from the 400px total.
-		
+
+		<img src="images/stripes-example-33-1.png" style="height:1em; width:400px;" alt="example 1">
```

Comment:
Maybe use and SVG instead of a PNG?

Please also avoid the example number in the name and just call it "stripes1", as the example number may change when other examples are added.

And if you want to provide an alternative text ‒ which is probably not needed as the output is already explained above the image ‒ it should provide a proper description of the image (see also https://www.w3.org/WAI/tutorials/images/tips/). So something like that:
"Horizontal line with a total width of 400 pixels showing the output of 'stripes(red 1fr, green 2fr, blue 100px)'. The left-most part is a red stripe of 100 pixels length, followed by a green stripe of 200 pixels, and a blue stripe of 100 pixels.".